test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -505,22 +505,39 @@ export default class PerfCompare extends Component {
                 curTestCI = parsedVariantsTest.metrics[j].value.CI;
 
                 // Get the metric's units
+                //Parse benchmark Variant and verify if the infos there
+                //get BenchmarkRouter & Metric files from server
+                let benchmarkMetricRequest = await fetch( `/api/getBenchmarkParsingInfo`, {
+                        method: 'get'
+                    }
+                );
+                let benchmarkMetricVariants = await benchmarkMetricRequest.json();
+                var BenchmarkMetricRouter = benchmarkMetricVariants['benchmarkMetricRouter'];
+                var BenchmarkMetric = benchmarkMetricVariants['benchmarkMetric'];
+                try {
+                    benchmarkProps = BenchmarkMetric[BenchmarkMetricRouter[curVariantData["benchmark"]][curVariantData["variant"]]]["metrics"];
+                } catch (routerNotFoundError) {
+                    //Either Benchmark that is parsed does not exist in the metric & router file or the name is different
+                    benchmarkProps = ""
+                    continue;
+                }
                 try {
-                    curMetricUnits = benchmarkVariantsInfo[curVariantData["benchmark"]][curVariantData["variant"]][curMetricName]["units"];
+                    curMetricUnits = benchmarkProps[curMetricName]["units"];
```

Comment:
We should be consistent with Tabular view, which uses `benchmarkProps["units"]`.